lib/client.ts

```diff
@@ -1,15 +1,16 @@
+import type { AxiosInstance } from "axios";
 import axios from "axios";
 import { version } from "../package.json";
 import * as envs from "./envs";
-import { Users } from "./users";
+import type * as magicLinks from "./magic_links";
 import { MagicLinks } from "./magic_links";
+import { OAuth } from "./oauth";
+import type * as otps from "./otps";
 import { OTPs } from "./otps";
 import { Sessions } from "./sessions";
-
-import type { AxiosInstance } from "axios";
 import type * as users from "./users";
-import type * as magicLinks from "./magic_links";
-import type * as otps from "./otps";
+import { Users } from "./users";
+
```

Comment:
I prefer to keep the `import type` statements separate from the regular `import` ones. Can you split them up again?

I'll see if I can get a Prettier or ESLint set up to enforce this in the future. Was this sorting done automatically by something?

Reply:
Yeah, vscode automatically sorts it on save but will revert.

Reply:
Interesting! I'll see if that's configurable when I investigate the formatter/linter further.